### PR TITLE
Added pull request template

### PR DESCRIPTION
**Pull request by DawidBastek**
## What does this PR do?

- Adds template for upcoming pull requests
- Updates Readme

## Required reviewers:
@marek-siemieniuk-morawski @aleparty

### CODE REVIEW

**Review by aleparty — APPROVED** [review]
lgtm

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## What does this PR do?
+*(A few words about the PR)*
```
Comment (marek-siemieniuk-morawski) [inline .github/pull_request_template.md:2]: I can't find any specific info, but I'm almost sure that you should add new lines. Check https://www.markdownguide.org/getting-started/ as a reference.

**Review by marek-siemieniuk-morawski — CHANGES_REQUESTED** [review]

**Comment by marek-siemieniuk-morawski** [comment]
Add others to PR as reviewers

File: .github/pull_request_template.md (hunk 1 above)
Comment (DawidBastek) [reply .github/pull_request_template.md:2]: Sounds reasonable, I will change that 👍

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Pull request merged**